package.json

```diff
@@ -98,5 +100,11 @@
   "bugs": {
     "url": "https://github.com/GoogleChrome/lighthouse/issues"
   },
+  "bundlesize": [
+    {
+      "path": "./lighthouse-extension/dist/scripts/lighthouse-background.js",
+      "threshold": "400 Kb"
```

Comment:
It's a gzipped size.

![image](https://user-images.githubusercontent.com/39191/28090408-feac2fa4-6640-11e7-9909-1f83ae755e1d.png)

right now it doesn't support non-gzipped size reporting.